src/components/common/animations/SpinnerCheckMark/SpinnerCheckMark.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Lottie from 'react-lottie'
+import animationData from './data.json'
+
+class SpinnerCheckMark extends React.Component {
+  constructor(props) {
+    super(props)
+    this.animationRef = React.createRef()
+  }
+
+  componentDidMount() {
+    this.animationRef.current.anim.onEnterFrame = e => {
```

Comment:
if you use https://github.com/felippenardi/lottie-react-web
you can use segments to define what to loop or play